scipy/stats/_unuran/meson.build

```diff
@@ -159,7 +164,7 @@ unuran_include_dirs = [
   '../../_lib/unuran/unuran/src/tests'
 ]
 
-unuran_version = '15:0:0'  # TODO: grab from configure.ac file
+unuran_version = '16:0:0'  # TODO: grab from configure.ac file
```

Comment:
Do you plan to change how the version is retrieved in this PR?

Reply:
I don't think it's worth trying to find this in the configure.ac file because (i) As you said, UNU.RAN doesn't get updated often and, (ii) It's not very inconvenient to just copy-paste the version from the file manually.